test/runtests.jl

```diff
@@ -95,6 +95,14 @@ module wrapper_chain
 	@from "chain.jl" import a, b, b2, c, c2, d, e
 end
 
+module wrapper_url
+	using FromFile
+	visible = [:A]
+	invisible = [:foo, :bar, :baz, :quux, :B, :C]
+
+	@from "https://cdn.jsdelivr.net/gh/Roger-luo/FromFile.jl@ba3d96b/test/basic.jl" import A
```

Comment:
Very cool to see that you're looking at URL support. This opens up all kinds of fun possibilities.

Nit: it'd be slightly tidier not to rely on another website (even an established one like jsDelivr), in favour of using something like `https://raw.githubusercontent.com/Roger-luo/FromFile.jl/master/src/FromFile.jl`.